agents/rust/mediator/mediation/src/router.rs

```diff
@@ -30,7 +27,7 @@ pub async fn create_router() -> Router {
             get(json::echo_message_json).post(respond_message_json),
         )
         .route("/forward", post(handle_forward))
-        .route("/pickup", post(handle_pickup))
+        // .route("/pickup", post(handle_pickup))
```

Comment:
what's the endpoint for client agent messages (connection, mediation, pickup) ?

Reply:
All client agent messages go through the `/didcomm` endpoint, which is part of the `mediator` src. 

Context: This router file is part of `mediation` crate (previously known as `xum-server`) and is basically dead code at this point. This router is unused by the `mediator`. We are only using the handler functions from this crate, some message structures and persistence functions. Not its server.